sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient_extensions.cs

```diff
@@ -98,199 +96,180 @@ public WebPubSubServiceClient(string connectionString, string hub) : this(ParseC
         }
 
         /// <summary>Broadcast message to all the connected client connections.</summary>
-        /// <param name="message"></param>
-        /// <param name="excluded">Excluded connection Ids.</param>
-        /// <param name="cancellationToken">The cancellation token to use.</param>
+        /// <param name="content"></param>
+        /// <param name="contentType">Defaults to ContentType.PlainText.</param>
         /// <returns>A <see cref="Response"/> if successful.</returns>
-        public virtual async Task<Response> SendToAllAsync(string message, IEnumerable<string> excluded = null, CancellationToken cancellationToken = default)
+        public virtual async Task<Response> SendToAllAsync(string content, ContentType contentType = default)
```

Comment:
I still miss having cancellation tokens on these high level methods, FWIW.

Reply:
Let's chat more about it. What I am going after is a very common BCL pattern where only the most powerful overload takes CTs. See https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.writeasync?view=net-5.0 as an example. 
I am not sure why we want to add CTs to all overloads.